From 0263f6f6712ef2682a8b8f4c8623df9f0612f5b5 Mon Sep 17 00:00:00 2001
From: Alin Jerpelea <alin.jerpelea@sony.com>
Date: Sat, 26 May 2018 09:29:44 +0200
Subject: [PATCH] display: fix undeclared variable

When DRM_IOCTL_SDE_WB_CONFIG is not declared the ret variable
is uninitialized and build is broken.

Test: manual - display build is fixed

Change-Id: Ic6e268acb5bbfbdb3ab66a597b1e28684ee63a0c
Signed-off-by: Alin Jerpelea <alin.jerpelea@sony.com>
---
 sdm845/sdm/libs/core/drm/hw_virtual_drm.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sdm845/sdm/libs/core/drm/hw_virtual_drm.cpp b/sdm845/sdm/libs/core/drm/hw_virtual_drm.cpp
index 5fe1d86c..90a736ec 100644
--- a/sdm845/sdm/libs/core/drm/hw_virtual_drm.cpp
+++ b/sdm845/sdm/libs/core/drm/hw_virtual_drm.cpp
@@ -95,12 +95,13 @@ void HWVirtualDRM::InitializeConfigs() {
   current_mode_.clock = (current_mode_.htotal * current_mode_.vtotal \
   * current_mode_.vrefresh) / 1000;
   struct sde_drm_wb_cfg wb_cfg;
+  int ret = 0;
   wb_cfg.connector_id = token_.conn_id;
   wb_cfg.flags |= SDE_DRM_WB_CFG_FLAGS_CONNECTED;
   wb_cfg.count_modes = 1;
   wb_cfg.modes = (uint64_t)&current_mode_;
   #ifdef DRM_IOCTL_SDE_WB_CONFIG
-  int ret = drmIoctl(dev_fd_, DRM_IOCTL_SDE_WB_CONFIG, &wb_cfg);
+  ret = drmIoctl(dev_fd_, DRM_IOCTL_SDE_WB_CONFIG, &wb_cfg);
   #endif
   if (ret) {
     DLOGE("WB config failed\n");
-- 
2.13.0

